src/test/java/selenium/LoadingDataDynamicallyTest.java

```diff
@@ -0,0 +1,36 @@
+package selenium;
+
+import lombok.extern.slf4j.Slf4j;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+import testbase.TestBase;
+
+import java.time.Duration;
+
+import static constants.ByConstants.GET_NEW_USER_BUTTON;
+import static constants.ByConstants.USER_INFORMATION_BOX;
+import static constants.TestConstants.LOADING_DATA_DYNAMICALLY_URL;
+import static constants.TimeOutConstants.MEDIUM_TIMEOUT;
+
+@Slf4j
+public class LoadingDataDynamicallyTest extends TestBase {
+
+    @Test
+    public void TestLoadingDataDynamically() {
```

Comment:
this test does not contain any validation, we should expect at least one condition which indicates that the test passed

Reply:
Added 3 assertions. Done!